api/services/Reddit.js

```diff
@@ -38,6 +38,9 @@ var makeRequest = async function (refreshToken, requestType, url, data, rateLimi
   if (left < rateLimitRemainingThreshold && moment().before(resetTime)) {
     throw {statusCode: 504, error: "Rate limited"};
   }
+  if (requestType === 'GET') {
+    url += (url.indexOf('?') === -1 ? '?' : '&') + 'raw_json=1';
+  }
```

Comment:
> Is there any reason to only run it on GET requests?

Not really. At first I thought that we only care about the response body for GET requests, but this was incorrect (e.g. we need a POST request to /api/flairselector to get someone's flair).